lib/omniauth/strategies/apple.rb

```diff
@@ -105,7 +119,7 @@ def verify_claims!(id_token)
         verify_aud!(id_token)
         verify_iat!(id_token)
         verify_exp!(id_token)
-        verify_nonce!(id_token) if id_token[:nonce_supported]
+        # verify_nonce!(id_token) if id_token[:nonce_supported]
```

Comment:
We need to figure out why this is done. We really shouldn't be disabling the nonce verification, as it would allow for replay attacks.

- https://forums.developer.apple.com/forums/thread/691220
- https://github.com/nhosoya/omniauth-apple/pull/111